### PR TITLE
added FileLengthCheck and ParameterNumberCheck

### PR DESCRIPTION
**Pull request by AlexHaxe**
Hi,

just added two more checks:
- FileLengthCheck: number of lines should be less than maxLines (default: 1000)
- ParameterNumberCheck: number of parameters per method should be less than maxParameters (default: 7)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,13 @@ More information in [wiki page](https://github.com/adireddy/haxe-checkstyle/wiki
 			}
 		},
 		{
+			"type": "FileLength",
+			"props": {
+				"severity": "WARNING",
+				"maxLength": 1000
```
Comment (adireddy) [inline README.md]: Can you please change `maxLength` to `max` similar to http://checkstyle.sourceforge.net/config_sizes.html

File: README.md [hunk 2]
```diff
@@ -102,6 +109,13 @@ More information in [wiki page](https://github.com/adireddy/haxe-checkstyle/wiki
 			}
 		},
 		{
+			"type": "ParameterNumber",
+			"props": {
+				"severity": "INFO",
+				"maxParameter": 7
```
Comment (adireddy) [inline README.md]: Same here. Please change `maxParameter` to `max` http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/sizes/ParameterNumberCheck.html

**Comment by adireddy** [comment]
Nice work :+1:

File: checkstyle/checks/FileLengthCheck.hx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import checkstyle.LintMessage.SeverityLevel;
+import haxeparser.Data.Token;
+
+@name("FileLength")
+@desc("Max number of lines per file (default 1000)")
+class FileLengthCheck extends Check {
+
+	public var severity:String = "WARNING";
+
+	public var maxLength:Int = 1000;
```
Comment (adireddy) [inline checkstyle/checks/FileLengthCheck.hx]: Can you please change `maxLength` to `max` similar to http://checkstyle.sourceforge.net/config_sizes.html

File: checkstyle/checks/ParameterNumberCheck.hx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+import checkstyle.LintMessage.SeverityLevel;
+import haxeparser.Data;
+import haxe.macro.Expr;
+
+@name("ParameterNumber")
+@desc("Max number of parameters per method (default 7)")
+class ParameterNumberCheck extends Check {
+
+	public var severity:String = "INFO";
+	public var maxParameter:Int = 7;
```
Comment (adireddy) [inline checkstyle/checks/ParameterNumberCheck.hx]: Same here. Please change `maxParameter` to `max` http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/sizes/ParameterNumberCheck.html

**Comment by adireddy** [comment]
:+1:  released to haxelib v1.0.9 http://lib.haxe.org/p/checkstyle

**Comment by AlexHaxe** [comment]
thanks!

I should have pointed it out in my commit message, but I changed the defaults to 2000 and 10, because the pages you linked me to were using those. I don't know if there are official defaults or whether every checkstyle implementation uses their own values. 
It's probably not a big deal since users can and will change the values according to their coding style, but the library should at least provide some reasonable defaults.

**Comment by adireddy** [comment]
FileLengthCheck default it 2000 and ParameterNumberCheck default is 7 as per the official checkstyle documentation.

Like you said it's not a big deal as long as they are configurable.

**Pull request merged**